package com.security.practice.Exception

fun main() {
    println("1$".toIntOrNull())
}